### PR TITLE
doc: security: Info for CVE 2023 4264

### PR DESCRIPTION
**Pull request by ceolin**
Add information about CVE-2023-4264

### CODE REVIEW

**Review by jhedberg — DISMISSED** [review]

**Review by nashif — DISMISSED** [review]

File: doc/security/vulnerabilities.rst [hunk 1]
```diff
@@ -1358,6 +1358,24 @@ This has been fixed in main for v3.5.0
...
+  <https://github.com/zephyrproject-rtos/zephyr/security/advisories/GHSA-rgx6-3w4j-gf5j>`_
+
+This has been fixed in main for v3.5.0
+
+- `PR 58834 fix for main
+  <https://github.com/zephyrproject-rtos/zephyr/pull/58834>`_
+- `PR 60465 fix for main
+  <https://github.com/zephyrproject-rtos/zephyr/pull/60465>`_
+- `PR 61845 fix for main
+  <https://github.com/zephyrproject-rtos/zephyr/pull/61845>`_
```
Comment (kartben) [inline doc/security/vulnerabilities.rst:1376]: Shouldn't backports be listed too?

File: doc/security/vulnerabilities.rst [hunk 2]
```diff
@@ -1358,6 +1358,24 @@ This has been fixed in main for v3.5.0
 - `PR 60079 fix for 3.3
   <https://github.com/zephyrproject-rtos/zephyr/pull/60079>`_
 
+CVE-2023-4264
+-------------
+
+Potential buffer overflow vulnerabilities in the Zephyr Bluetooth subsystem
+
+- `Zephyr project bug tracker GHSA-rgx6-3w4j-gf5j
+  <https://github.com/zephyrproject-rtos/zephyr/security/advisories/GHSA-rgx6-3w4j-gf5j>`_
```
Comment (kartben) [inline doc/security/vulnerabilities.rst:1367]: it looks like the advisory only captures #60465 in the list of patches? I don't think that's right as we want people digging into the CVE from someplace else (ex. NIST database) to easily be able to access complete info?

**Review by kartben — CHANGES_REQUESTED** [review]

File: doc/security/vulnerabilities.rst (hunk 1 above)
Comment (ceolin) [reply doc/security/vulnerabilities.rst:1376]: They were not backported

File: doc/security/vulnerabilities.rst (hunk 2 above)
Comment (ceolin) [reply doc/security/vulnerabilities.rst:1367]: Yep you are right. I've updated the advisory.

**Comment by ceolin** [comment]
@kartben AFAICT there is nothing to change in this pr. Have I missed something ?

**Comment by ceolin** [comment]
pr rebased.

**Review by jhedberg — DISMISSED** [review]

File: doc/security/vulnerabilities.rst (hunk 1 above)
Comment (kartben) [reply doc/security/vulnerabilities.rst:1386]: looks like at least one was (so maybe others should be to, at least for 3.4)? https://github.com/zephyrproject-rtos/zephyr/pull/61385

**Comment by kartben** [comment]
> @kartben AFAICT there is nothing to change in this pr. Have I missed something ?

see my latest comment, at least one backport is missing.

File: doc/security/vulnerabilities.rst (hunk 1 above)
Comment (ceolin) [reply doc/security/vulnerabilities.rst]: oh, thanks for pointing it out.

**Comment by ceolin** [comment]
> > @kartben AFAICT there is nothing to change in this pr. Have I missed something ?
> 
> see my latest comment, at least one backport is missing.

thanks ! Added the backported pr.

**Review by kartben — APPROVED** [review]
Thanks

**Review by jhedberg — APPROVED** [review]

**Pull request merged**